src/modules/player/player.ts

```diff
@@ -127,6 +127,25 @@ export class Player {
                 });
             });
 
+            client.on('player_update_busy', function (payload) {
+                if (client.playerData === undefined) {
+                    return;
+                }
+                if (client.playerData.isBusy === payload) {
```

Comment:
the property was updated to `playerData.status`

Reply:
Lies!